pkgs/development/libraries/qt-5/qtModule.nix

```diff
@@ -34,13 +34,15 @@ mkDerivation (args // {
     fixQtBuiltinPaths . '*.pr?'
   '';
 
+  # Note: The root include path is missing and is added here (see #52457).
   postFixup = ''
     if [ -d "''${!outputDev}/lib/pkgconfig" ]; then
         find "''${!outputDev}/lib/pkgconfig" -name '*.pc' | while read pc; do
             sed -i "$pc" \
                 -e "/^prefix=/ c prefix=''${!outputLib}" \
                 -e "/^exec_prefix=/ c exec_prefix=''${!outputBin}" \
-                -e "/^includedir=/ c includedir=''${!outputDev}/include"
+                -e "/^includedir=/ c includedir=''${!outputDev}/include" \
```

Comment:
This is werid. Multiple outputs setup hook should already handle this:

https://github.com/NixOS/nixpkgs/blob/f3ce7c5582c1a71baf133e6e85e8517e162abc50/pkgs/build-support/setup-hooks/multiple-outputs.sh#L161-L164

Reply:
That change in `qtModule.nix` is [3 years old](https://github.com/NixOS/nixpkgs/commit/fe0ab944db2176e6757ae89aeebea16ab8dc69d0#diff-88c589c484358b9563a1084ccf34fb69R43), so maybe its superfluous now. 

I made a quick test and the results does indeed seem fine without it. It's however unrelated to what I'm trying to accomplish.